spells.yml

```diff
@@ -3604,6 +3604,32 @@ fey_mischief:
     - illusion
     - sorcerer
     - wizard
+fiery_whip:
+  casting_time: 1 minor action
+  components: V, S
+  concentration: true
+  description: |-
+    A whip of fire appears in your free hand. The whip is a weapon with the finesse, light, and reach 15 properties. On a hit, the whip deals 2d4 fire damage, and if the target is no more than one size larger than you, you can pull it up to 5 feet towards you.
```

Comment:
How would you feel about moving them up to 10 feet towards you? Could be interesting then if you're dual wielding, drag the creature to within 5 feet of you with one attack, and then hit them with your other weapon for the other attack.